Dockerfile

```diff
@@ -37,12 +37,11 @@ RUN case ${TARGETPLATFORM} in \
     GOARCH=${GOARCH} VERSION=${VERSION} make build && touch jwt_signing_key.pem
 
 # Copy binary to alpine
-FROM alpine:3.15
+FROM gcr.io/distroless/static-debian11:nonroot
```

Comment:
I would expect to publish a separate image so that we maintain the old alpine image for those that rely on it. Is there a way we can parameterise this?

Reply:
There is, I will take a look and update the PR

Reply:
I've parameterised the runtime image, so it's possible to adjust which image to be used in the make file.

Reply:
Which would you prefer to be default, distroless or alpine image?

Reply:
Just a question, is it necessary to create a separate tag for different platforms? The `$(DOCKER_BUILDX_X_PLATFORM) -f Dockerfile -t $(REGISTRY)/oauth2-proxy:${VERSION} .` does cover all of them in one

Reply:
At the moment, we can only push one version right? Perhaps we need a separate target that pushes a set of distroless tags?

For now I think the default should remain as alpine so that it's not a surprising change to users